### PR TITLE
Revert changes from PR #251

### PR DESCRIPTION
**Pull request by awoie**
This PR reverts changes introduced in PR #251

### CODE REVIEW

**Review by peacekeeper — APPROVED** [review]

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 1]
```diff
@@ -1534,12 +1571,11 @@ for their contributions (some of which substantial) to this draft and to the ini
 # Document History
 
 -07
-
+* Add guidance on using DIDs for issuer key discovery
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:1574]: This should make clear that it reverts a previous change.

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 2]
```diff
@@ -1534,12 +1571,11 @@ for their contributions (some of which substantial) to this draft and to the ini
 # Document History
 
 -07
-
+* Revert change from previous release
 
 -06
 
 * Update the anticipated media type registration request from `application/vc+sd-jwt` to `application/dc+sd-jwt`
-* Tightened the exposition of the Issuer-signed JWT Verification Key Validation section
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:1542]: This should remain in the list.

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 3]
```diff
@@ -1534,12 +1571,11 @@ for their contributions (some of which substantial) to this draft and to the ini
 # Document History
 
 -07
-
+* Revert change from previous release
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:1574]:
```suggestion
* Revert change from previous release that removed explicit mention of DIDs in the Issuer-signed JWT Verification Key Validation section
```

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 4]
```diff
@@ -1534,12 +1571,11 @@ for their contributions (some of which substantial) to this draft and to the ini
 # Document History
 
 -07
-
+* Revert change from previous release that removed explicit mention of DIDs in the Issuer-signed JWT Verification Key Validation section
 
 -06
 
 * Update the anticipated media type registration request from `application/vc+sd-jwt` to `application/dc+sd-jwt`
```
Comment (awoie) [inline draft-ietf-oauth-sd-jwt-vc.md:1578]:
```suggestion
* Update the anticipated media type registration request from `application/vc+sd-jwt` to `application/dc+sd-jwt`
* Tightened the exposition of the Issuer-signed JWT Verification Key Validation section
```

**Review by bc-pi — APPROVED** [review]
under duress

**Pull request merged**